aiocache/decorators.py

```diff
@@ -72,14 +72,19 @@ def __call__(self, f):
         return wrapper
 
     async def decorator(self, f, *args, **kwargs):
+        disable_read = kwargs.pop('aiocache_disable_read', False)
+        disable_write = kwargs.pop('aiocache_disable_write', False)
```

Comment:
Put explicitly in the `function` signature please

Reply:
Also, add docs about those two new parameters in the class docstring or in `decorators.rst`

Reply:
Done